migrations/sql/V2020.08.25.13.45__add_draft_status_to_permit_status.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO permit_status_code
+    (
+    permit_status_code,
+    description,
+    display_order,
+    create_user,
+    update_user
+    )
+VALUES
+    ('D', 'Draft', 20, 'system-mds', 'system-mds')
+ON CONFLICT DO NOTHING;
```

Comment:
This was already added in the 2020.07.09 draft_permit sql script

Reply:
Okay, I didn't notice that. 
I think it was being overwritten by the script in the seed file, I wasn't able to see this status in my db thats why I added it. 

I will remove this file